overview/content-types/commands.nix

```diff
@@ -9,10 +9,15 @@ let
   # TODO: wrap in submodule and make it render itself
   commands = {
     bash = mkOption {
-      type = types.submodule ./bash-code.nix;
+      type = with types; nullOr (submodule ./bash-code.nix);
+      default = null;
```

Comment:
```suggestion
      type = with types; listOf (submodule ./bash-code.nix);
      default = [];
```

Reply:
I think it would be best to discuss this more (maybe tomorrow) for what's the best way to represent the commands and then we can make a follow-up PR that implements that.